code/index.html

```diff
@@ -2,29 +2,117 @@
 
-    <!-- Signup form -->
+  <body>
+    
+    <!-- HERO BACKGROUND -->
+    <header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <!-- HERO TEXT CONTAINER -->
+      <div class="flexParentHeader">
+        <div class="headerText">
+          Pepped Web Agency
+        </div>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+        <nav class="navbar">
+          <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Our services</a></li>
+            <li><a href="#accordion">FAQ</a></li>
+          </ul>
+        </nav>
       </div>
+     
+    </header>
+
+    <!-- MAIN CONTENT HERE -->
+      <main>
+        <div class="mainWrapper">
+
+          <!-- CONTACT FORM -->
+            <div class="formWrapper">
+              <section id="form">
+                <form method="POST" action="http://httpbin.org/anything">
+                  
+                  <h2>Need a pep talk?</h2>
+
+                  <!-- TEXT INPUTS -->
+                  <label class="blockLabel" for="name">Name</label>
+                  <input class="blockInput" type="text" id="name" name="name" placeholder="Your name..."required/>
+
+                  <label class="blockLabel" for="email">Email</label>
+                  <input class="blockInput" type="email" id="email" name="email" placeholder="Your email..."/>
+
+                  <label class="blockLabel" for="number">Number</label>
+                  <input class="blockInput" type="number" id="number" name="number" placeholder="Your number..."/>
+                  
+                  <label class="blockLabel" for="services">Services needed</label>
+                    <select class="blockInput" name="services" id="services">
+                      <option>Web Development</option>
+                      <option>Content Creation</option>
+                      <option>Pepped Coaching</option>
+                    </select>
+
+                  <label class="blockLabel" for="message">Message</label>
+                  <textarea class="blockInput" name="message" id="message" cols="20" rows="10">Tell us your story!</textarea> 
+
+                  <button id="button" type="submit">Get pepped</button>
+
+                </form>
+              </section>
+            </div>
+      
+      
+          <!-- ACCORDION CONTAINER -->
+          <div class="accordionWrapper">
+            <section class="accordion" id="accordion">
+             
+              <h2>FAQ</h2>
+
+              <div id="questionBox">  
+                <div class="question" id="question1">Why should I hire Pepped?</div>
+                <div class="answer" id="answer">
+                <p>First of all, by the sound of our name - we guarantee that you will get more pepp in your life.</p>
+                </div>
+              </div>
+
+              <div id="questionBox">
```

Comment:
Maybe it would be better to use class instead if Id for `#questionbox`? I think you are only supposed to use id when its one specifik element.